pwa/src/styling/design-tokens/conduction-design-tokens.css

```diff
@@ -0,0 +1,4 @@
+:root {
+  --conduction-nav-logo-height-lg: 100px;
+  --conduction-nav-logo-padding: 10px;
```

Comment:
When we have `--utrecht` variables available, we can best use them for consistency. Creating our own design tokens is only meant as a last-resort solution.

(Utrecht has tokens for 12px and 8px, each of those would've been close enough, I think).